gcf/custom/manifest.go

```diff
@@ -48,6 +48,58 @@ func computeDataMCF(bucket, root, im, tab string, mcf []string) []string {
 	return bigstoreMCFs
 }
 
+func computeSource(
+	ctx context.Context, reader Reader, bucket, root, provFile string,
+) (
+	*pb.DataCommonsManifest_DatasetSource, error,
+) {
+	// Populate data source that should be specified at import group level.
+	if provFile != "" {
+		provJSON, err := reader.ReadObject(
+			ctx, bucket, path.Join(root, "data", provFile))
```

Comment:
As per [doc](https://docs.datacommons.org/custom_dc/upload_data.html), should this be path.Join(root, provFile)?

Reply:
good catch, the doc needs to be updated, "data" folders should have all the user data